docker-entrypoint

```diff
@@ -45,7 +46,13 @@ get_container_info() {
   else
     label_filter=",\"label\":\[\"${AUTOHEAL_CONTAINER_LABEL}=true\"\]"
   fi
-  url="${HTTP_ENDPOINT}/containers/json?filters=\{\"health\":\[\"unhealthy\"\]${label_filter}\}"
+  if [ "$AUTOHEAL_ONLY_MONITOR_RUNNING" = false ]
+  then
+    running_filter=""
+  else
+    running_filter=",\"status\":\[\"runing\"\]"
```

Comment:
Shouldn't this be `"running"`?

```suggestion
    running_filter=",\"status\":\[\"running\"\]"
```

Reply:
Yeah, I spotted this and changed it in my main branch, but I've made some other changes in there which won't want merging upstream. I thought I'd edited it in this branch too, but missed it. Should all be good now.